### PR TITLE
Add missing project permissions on appcat service namespace

### PR DESCRIPTION
**Pull request by TheBigLee**
In order for the end user to properly see the logs in the console, the project permission is required.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

File: component/main.jsonnet [hunk 1]
```diff
@@ -70,6 +70,11 @@ local readServices = kube.ClusterRole('appcat:services:read') + {
       resources: [ 'pods/portforward' ],
       verbs: [ 'get', 'list', 'create' ],
     },
+    {
+      apiGroups: [ '""', 'project.openshift.io' ],
```
Comment (glrf) [inline component/main.jsonnet:74]: `'""'` seems wrong/unnecessary. Why did you add that?

File: apis/generated/appcat.vshn.io_objectbuckets.yaml
Comment (glrf) [inline apis/generated/appcat.vshn.io_objectbuckets.yaml:1]: I don't think we commit the `apis` folder anymore. We should probably add that to the gitignore

**Review by glrf — APPROVED** [review]
TIL

Please fix the golden tests and remove the `apis` directory. Otherwise looks good to me

File: component/main.jsonnet (hunk 1 above)
Comment (TheBigLee) [reply component/main.jsonnet:74]: It is specified like this in the default `view` clusterRole. Not sure why though

File: apis/generated/appcat.vshn.io_objectbuckets.yaml
Comment (TheBigLee) [reply apis/generated/appcat.vshn.io_objectbuckets.yaml:1]: Removed them again

File: component/main.jsonnet (hunk 1 above)
Comment (glrf) [reply component/main.jsonnet:74]: Hmm interesting. Maybe some OpenShift weirdness/backwards compatibility thing?

But also to get the same output you'll need to use 

```suggestion
      apiGroups: [ '', 'project.openshift.io' ],
```

**Review by simu — APPROVED** [review]
